src/stream.rs

```diff
@@ -6,28 +6,30 @@ use std::slice;
 
 use crate::eq::EqKey;
 use crate::le::LeKey;
-use crate::utils::MMO;
+use crate::utils::Mmo;
 use crate::N;
 
 pub trait FSSKey: Sized {
-    fn eval(&self, prg: &mut impl PRG, party_id: u8, x: u32) -> u32;
+    fn eval(&self, prg: &mut impl Prg, party_id: u8, x: u32) -> u32;
 
-    fn generate_keypair(prg: &mut impl PRG) -> (Self, Self);
+    fn generate_keypair(prg: &mut impl Prg) -> (Self, Self);
 }
 
 pub trait RawKey: Sized {
     const KEY_LEN: usize;
 
+    /// # Safety Call unsafe function internally
     unsafe fn from_raw_line(raw_line_pointer: *const u8) -> Self;
```

Comment:
I think it would be more appropriate here to say that we are de-referencing a raw pointer. See [this](https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html#dereferencing-a-raw-pointer) for example.

Reply:
Mmm. Then I guess that I should change it in 4 places right?

Reply:
Yes, I think it makes sense.  What do you think?

Reply:
Agree with @arturomf94 :)

Reply:
Agree with both of you of course. My explanation could be true or false. But yours is always true!! Thanks!!!